insonmnia/miner/plugin/plugin.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/api/types/network"
 	log "github.com/noxiouz/zapctx/ctxlog"
+	"github.com/pkg/errors"
```

Comment:
`fmt.Errorf`?

Reply:
still don't understand why fmt package is responsible for error creation, but ok